src/test/java/org/folio/rest/api/TestBase.java

```diff
@@ -89,108 +184,14 @@ public static <T> T get(Future<T> future) {
-    inventoryViewClient = ResourceClient.forInventoryView(client);
-    statisticalCodeClient = ResourceClient.forStatisticalCodes(client);
-    instancesStorageBatchInstancesClient = ResourceClient
-      .forInstancesStorageBatchInstances(client);
-    instanceTypesClient = ResourceClient
-      .forInstanceTypes(client);
-    illPoliciesClient = ResourceClient.forIllPolicies(client);
-    statisticalCodeFixture = new StatisticalCodeFixture(client);
-    kafkaConsumer = new FakeKafkaConsumer().consume(vertx);
-    kafkaConsumer.removeAllEvents();
-    instanceReindex = new InstanceReindexFixture(client);
-    authorityReindex = new AuthorityReindexFixture(client);
-    asyncMigration = new AsyncMigrationFixture(client);
-    logger.info("finishing @BeforeClass testBaseBeforeClass()");
-  }
-
-  @AfterClass
-  public static void testBaseAfterClass()
-    throws InterruptedException,
-    ExecutionException,
-    TimeoutException {
-
-    client.getWebClient().close();
-    client = null;
-    if (invokeStorageTestSuiteAfter) {
-      StorageTestSuite.after();
-    }
-  }
-
-  static void send(URL url, HttpMethod method, String content,
-      String contentType, Handler<HttpResponse<Buffer>> handler) {
-    send(url, method, null, content, contentType, handler);
-  }
-
-  static void send(URL url, HttpMethod method, String userId, String content,
-                   String contentType, Handler<HttpResponse<Buffer>> handler) {
-    send(url.toString(), method, userId, content, contentType, handler);
-  }
-
-  static Future<HttpResponse<Buffer>> send(URL url, HttpMethod method, String content,
-      String contentType) {
-    return Future.future(promise -> send(url, method, content, contentType, promise::complete));
-  }
-
-  static void send(String url, HttpMethod method, String content,
-                   String contentType, Handler<HttpResponse<Buffer>> handler) {
-    send(url, method, null, content, contentType, handler);
-  }
-
-  static void send(String url, HttpMethod method, String userId, String content,
-        String contentType, Handler<HttpResponse<Buffer>> handler) {
-    Buffer body = Buffer.buffer(content == null ? "" : content);
-
-    MultiMap headers = MultiMap.caseInsensitiveMultiMap();
-    if (userId != null) {
-      headers.add("X-Okapi-User-Id", userId);
-    }
-    client.getWebClient()
-    .requestAbs(method, url)
-    .putHeader("Authorization", "test_tenant")
-    .putHeader("x-okapi-tenant", "test_tenant")
-    .putHeader("Accept", "application/json,text/plain")
-    .putHeader("Content-type", contentType)
-    .putHeaders(headers)
-    .sendBuffer(body)
-    .onSuccess(handler)
-    .onFailure(error -> logger.error(error.getMessage(), error));
-  }
-
   /**
    * Assert that a GET at the url returns 404 status code (= not found).
    * @param url  endpoint where to execute a GET request
    */
-  void assertGetNotFound(URL url) {
+  public void assertGetNotFound(URL url) {
```

Comment:
Why does this need to be public? Is it being used by something outside of TestBase?

Reply:
One of my quirks where I strive for consistency.
It just seems that the many other methods in the test classes are generally `public`.

I can remove the `public`.

Reply:
> It just seems that the many other methods in the test classes are generally public.

I suspect that's because folks were using the suites / bases as places to put stuff for reuse even when not inheriting.